data/json/vehicle_parts.json

```diff
@@ -5157,7 +5160,7 @@
         "item" : "atomic_lamp",
         "difficulty" : 1,
         "location" : "on_ceiling",
-        "flags": ["ATOMIC_LIGHT", "LEAK_DAM", "RADIOACTIVE"],
+        "flags": ["ATOMIC_LIGHT", "LEAK_DAM", "RADIOACTIVE", "TOOL_SCREWDRIVER"],
```

Comment:
Someone did it b/c it's atomic :P Can't recall the PR but it was fairly recently.